### PR TITLE
[office-js][office-js-preview] (Excel) Pulling latest Excel type definitions from CDN, releasing ExcelApi 1.15 to GA

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
Release URL: https://appsforoffice.microsoft.com/lib/1.1/hosted/office.debug.js
Beta URL: https://appsforoffice.microsoft.com/lib/beta/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -37651,22 +37682,6 @@ declare namespace Excel {
     class ChartFill extends OfficeExtension.ClientObject {
         /** The request context associated with the object. This connects the add-in's process to the Office host application's process. */
         context: RequestContext;
-        /**
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:37654]: @wandyezj This block is removed from office-js-preview in the most recent CDN update, can you confirm it should be removed?

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -37683,32 +37698,11 @@ declare namespace Excel {
          * @param color HTML color code representing the color of the background, in the form #RRGGBB (e.g., "FFA500") or as a named HTML color (e.g., "orange").
          */
         setSolidColor(color: string): void;
-        /**
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:37686]: @wandyezj This block is removed from office-js-preview in the most recent CDN update, can you confirm it should be removed? 

`toJSON(): Excel.Interfaces.ChartFillData;` is not removed, so removing this block results in an npm lint error.

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -61977,17 +61970,6 @@ declare namespace Excel {
             */
             font?: Excel.Interfaces.ChartFontUpdateData;
         }
-        /** An interface for updating data on the ChartFill object, for use in `chartFill.set({ ... })`. */
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:61980]: @wandyezj Please check this as well

File: types/office-js-preview/index.d.ts [hunk 4]
```diff
@@ -68215,17 +68197,6 @@ declare namespace Excel {
             */
             font?: Excel.Interfaces.ChartFontData;
         }
-        /** An interface describing the data returned by calling `chartFill.toJSON()`. */
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:68218]: @wandyezj Please check this as well

File: types/office-js-preview/index.d.ts [hunk 5]
```diff
@@ -77352,26 +77322,6 @@ declare namespace Excel {
             */
             font?: Excel.Interfaces.ChartFontLoadOptions;
         }
-        /**
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:77355]: @wandyezj Please check this as well

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (wandyezj) [reply types/office-js-preview/index.d.ts:37654]: It's ok for beta APIs to be removed from the office-js-preview.

File: types/office-js/index.d.ts [hunk 6]
```diff
@@ -19691,6 +19691,44 @@ declare namespace Excel {
...
+        /**
+         * The data source type is unknown or unsupported.
+         * @remarks
+         * [Api set: ExcelApi 1.15]
+         */
+        unknown = "Unknown",
+        /**
+         * Contains a cube name for an OLAP data source.
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
```
Comment (alison-mk) [inline types/office-js/index.d.ts:19710]: @wandyezj Why are there preview APIs in office-js? Here and line 19728

Comment (alison-mk) [reply types/office-js/index.d.ts:19710]: (Removing per internal conversation)

Comment (wandyezj) [reply types/office-js/index.d.ts:19710]: Those should be removed.

**Review by samantharamon — APPROVED** [review]
Approve

**Pull request merged**